apps/lockfile-explorer/src/commands/check.ts

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import type { Lockfile, LockfileV6 } from '@pnpm/lockfile-types';
+import path from 'path';
+import yaml from 'js-yaml';
+import { RushConfiguration } from '@microsoft/rush-lib/lib/api/RushConfiguration';
+import type { Subspace } from '@microsoft/rush-lib/lib/api/Subspace';
+import type { RushConfigurationProject } from '@microsoft/rush-lib/lib/api/RushConfigurationProject';
+import { FileSystem } from '@rushstack/node-core-library';
+import type { CommandModule } from 'yargs';
```

Comment:
Could you add a test that makes a Jest snapshot of the CLI help? Otherwise it's difficult for code reviewers to review it.

```
lockfile-explorer [command]

Commands:
  lockfile-explorer check                   Check if the specified package has a
  <package-specifier>                       inconsistent package versions in
                                            target project
  lockfile-explorer start                   Start the application

Options:
  -h, --help     Show help                                             [boolean]
      --version  Show version number                                   [boolean]
```

```
lockfile-explorer check <package-specifier>

Check if the specified package has a inconsistent package versions in target
project

Positionals:
  packageSpecifier  The name of the package and version to check        [string]

Options:
  -h, --help     Show help                                             [boolean]
      --version  Show version number                                   [boolean]
      --project  The name of the project that should be checked         [string]
```

Reply:
done😊